### PR TITLE
Add a separate emulator disabled card for extensions

### PR DESCRIPTION
**Pull request by joehan**
@markarndt noticed that the EmulatorDisabled component shows instructions that don't apply for the Extensions emulator, so I added a new component with correct instructions.

Finally got back to this and got it running locally:
Screen Shot 2022-06-28 at 4 56 23 PM

### CODE REVIEW

**Review by yuchenshi — APPROVED** [review]

File: src/components/common/EmulatorDisabled.tsx [hunk 1]
```diff
@@ -59,6 +59,43 @@ export const EmulatorDisabled: React.FC<
...
+  React.PropsWithChildren<{}>
+> = () => (
+  <>
+    <GridCell desktop={2} tablet={0} phone={0}>
+      {/* Spacing */}
+    </GridCell>
+    <GridCell span={8} align="middle">
+      <Card className="EmulatorDisabled">
+        <div className="EmulatorDisabled-graphics">
+          <img src="/assets/img/database.png" alt="" />
```
[the remark this answers is not in the document]
Comment (joehan) [reply src/components/common/EmulatorDisabled.tsx]: Good point - swapped it for the default extensions logo image.

**Review by markarndt — APPROVED** [review]

**Review by tjlav5 — DISMISSED** [review]

File: src/components/common/EmulatorDisabled.tsx [hunk 2]
```diff
@@ -59,6 +59,43 @@ export const EmulatorDisabled: React.FC<
...
+            If you want to use Extensions in your testing, add them to your
+            extension manifest. To do so, either to add a single extension
+            instance by running:
+            <code>firebase ext:install &gt;publisherId/extensionId&lt;</code>
+            or export the extensions instances from a production project by
+            running
+            <code>firebase ext:export --project=your-project</code>.
+          </Typography>
+          <Typography use="body1" tag="p">
+            <a href="https://firebase.google.com/docs/extensions/manifest">
```
Comment (NothingEverHappens) [inline src/components/common/EmulatorDisabled.tsx:89]: There is a docs link component somewhere which takes care of the looks

File: src/components/common/EmulatorDisabled.tsx [hunk 3]
```diff
@@ -59,6 +59,43 @@ export const EmulatorDisabled: React.FC<
   </>
 );
 
+export const ExtensionsEmulatorDisabled: React.FC<
+  React.PropsWithChildren<{}>
```
Comment (NothingEverHappens) [inline src/components/common/EmulatorDisabled.tsx:63]: `React.PropsWithChildren<{}>` this should be safe to drop?

File: src/components/common/EmulatorDisabled.tsx [hunk 4]
```diff
@@ -59,6 +59,43 @@ export const EmulatorDisabled: React.FC<
...
+            extension manifest. To do so, either to add a single extension
+            instance by running:
+            <code>firebase ext:install &gt;publisherId/extensionId&lt;</code>
+            or export the extensions instances from a production project by
+            running
+            <code>firebase ext:export --project=your-project</code>.
+          </Typography>
+          <Typography use="body1" tag="p">
+            <a href="https://firebase.google.com/docs/extensions/manifest">
+              <HintIcon size="xsmall" /> Learn more
```
Comment (NothingEverHappens) [inline src/components/common/EmulatorDisabled.tsx:90]: Consider phrasing this better, see [go/meaningful-links](https://goto.google.com/meaningful-links)

**Review by NothingEverHappens — APPROVED** [review]

File: src/components/common/EmulatorDisabled.tsx (hunk 3 above)
Comment (joehan) [reply src/components/common/EmulatorDisabled.tsx]: Removed

File: src/components/common/EmulatorDisabled.tsx (hunk 4 above)
Comment (joehan) [reply src/components/common/EmulatorDisabled.tsx]: Good call - ended up moving this link inline into the text to be more meanigful

File: src/components/common/EmulatorDisabled.tsx (hunk 2 above)
Comment (joehan) [reply src/components/common/EmulatorDisabled.tsx:79]: Removed this to inline it instead

**Pull request merged**